src/server/bootstrap.ts

```diff
@@ -279,10 +281,9 @@ export async function bootstrap() {
     if ((config.get('GIVING_BLOCKS_SERVICE_ACTIVE') as string) === 'true') {
       runGivingBlocksProjectSynchronization();
     }
-    await getCsvAirdropTransactions(
-      '0x0c452a7c116adb6162390f342cee84175f34e3c1bc0015e6f82773a54ace3061',
-      100,
-    );
```

Comment:
It was for test when implementing rainbow roll donations, but I forgot to delete it that time